examples/sabre_command_executor/src/main.rs

```diff
@@ -0,0 +1,560 @@
+        value: admin_keys_setting_bytes,
+    })
+}
+
+fn get_state_change(result: BatchExecutionResult) -> Vec<(String, Vec<u8>)> {
+    let results = get_result(result);
+
+    let mut state_change_pairs = Vec::new();
+
+    for state_changes in results {
+        for state_change in state_changes {
+            match state_change {
+                StateChange::Set { key, value } => state_change_pairs.push((key, value)),
+                StateChange::Delete { key } => state_change_pairs.push((key, vec![])),
+            }
+        }
+    }
+
+    state_change_pairs
+}
+
+fn get_result(batch_result: BatchExecutionResult) -> Vec<Vec<StateChange>> {
+    let mut results = Vec::new();
+
+    for receipt in batch_result.receipts {
+        match receipt.transaction_result {
+            TransactionResult::Valid { state_changes, .. } => results.push(state_changes),
+            TransactionResult::Invalid { error_message, .. } => {
+                // Check to see if the invalid transaction result is expected because
+                // one of the possible transactions created by the workload is RETURN_INVALID
+                if error_message
+                    .contains("Wasm contract returned invalid transaction: command, 1.0")
+                {
+                    continue;
+                } else {
+                    panic!("Transaction failed: {:?}", error_message)
+                }
+            }
+        }
+    }
+    results
+}
+
+fn create_executor(
+    context_manager: &ContextManager,
+) -> Result<Executor, SabreCommandExecutorError> {
+    Ok(Executor::new(vec![Box::new(
+        StaticExecutionAdapter::new_adapter(
+            vec![Box::new(SawtoothToTransactHandlerAdapter::new(
+                SabreTransactionHandler::new(),
+            ))],
+            context_manager.clone(),
+        )
+        .map_err(|err| {
+            SabreCommandExecutorError::Internal(format!(
+                "Unable to create StaticExecutionAdapter: {}",
+                err
+            ))
+        })?,
+    )]))
+}
+
+fn create_signer() -> Box<dyn Signer> {
+    let context = Secp256k1Context::new();
+    let key = context.new_random_private_key();
+    context.new_signer(key)
+}
+
+fn execute_transaction(
+    task_executor: &ExecutionTaskSubmitter,
+    context_lifecycle: Box<dyn ContextLifecycle>,
+    state_root: &str,
+    batch_pair: BatchPair,
+) -> Result<BatchExecutionResult, SabreCommandExecutorError> {
+    let mut scheduler =
+        SerialScheduler::new(context_lifecycle, state_root.to_string()).map_err(|err| {
+            SabreCommandExecutorError::Internal(format!("Unable to create scheduler: {}", err))
+        })?;
+
+    let (result_sender, result_receiver) = std::sync::mpsc::channel();
+
+    scheduler
+        .set_result_callback(Box::new(move |batch_result| {
+            result_sender
+                .send(batch_result)
+                .expect("Unable to send batch result")
```

Comment:
Remove the expects in this function - if this particular call fails, it should log an error.